vignettes/setup/setup.R

```diff
@@ -21,15 +21,12 @@ knit_print.grViz <- function(x, ...) {
 
 # If input loads dm or tidyverse, we load it here to omit warnings
 input <- readLines(knitr::current_input())
-if (rlang::has_length(grep("^library[(][?\"]dm[?\"][)]", input))) {
+if (rlang::has_length(grep("^library[(]dm[)]", input))) {
```

Comment:
```suggestion
if (rlang::has_length(grep('^library[(]"?dm"?[)]', input))) {
```

It's not perfect but does the job.

Reply:
Thanks. Before I commit this, what do you think about the comment [here](https://github.com/cynkra/dm/pull/1098#discussion_r903302715)?

Reply:
I'm fine either way. If we want to be strict, we could detect `libary("xxx")` calls here, and fail?

Reply:
Failing feels a bit extreme to me. 

Okay, let's not invest so much energy into this, since none of the collaborators might actually end up calling `library()` it this way. So we can support both ways of calling it, but ideally suggest the preferred alternative during the code review.